### ISSUE TITLE
oc rsh doesnt work after install

### ISSUE THREAD

**Issue by karmab** [openshift-metal3/dev-scripts#855]
**Describe the bug**
After openshift is deployed using baremetal ipi (no dev scripts though), oc rsh doesnt work, since it seems to use the provisioning ip for this matter.

**To Reproduce**
Steps to reproduce the behavior, please include *all* details about the dev-scripts version (git commit SHA), any local variable overrides or other customizations, and whether you're deploying on VMs or Baremetal.
- deploy baremetal ipi
- try to oc rsh in one of the pod

**Expected/observed behavior**

```
error: unable to upgrade connection: error dialing backend: dial tcp 172.22.0.37:10010: connect: no route to host
```
**Additional context**

**Comment by karmab** [comment]
please note that it only affects pods with host: network

**Comment by e-minguez** [comment]
It happens to me as well, but only in the master hosts (despite the names of the workers are 'masters'):

```
$ oc get nodes
NAME                                         STATUS   ROLES    AGE     VERSION
kni1-master-0.example.com   Ready    master   3h39m   v1.16.2
kni1-master-1.example.com   Ready    worker   176m    v1.16.2
kni1-master-2.example.com   Ready    worker   174m    v1.16.2
kni1-worker-0.example.com   Ready    worker   173m    v1.16.2

$ oc new-project test
$ oc create deployment hello-node --image=gcr.io/hello-minikube-zero-install/hello-node
$ oc scale deployment/hello-node --replicas=4
$ oc get po -o wide
NAME                          READY   STATUS    RESTARTS   AGE     IP            NODE                                         NOMINATED NODE   READINESS GATES
hello-node-7676b5fb8d-7xm8s   1/1     Running   0          6m21s   10.128.0.87   kni1-master-0.example.com   <none>           <none>
hello-node-7676b5fb8d-g76bk   1/1     Running   0          4m21s   10.130.0.6    kni1-master-2.example.com   <none>           <none>
hello-node-7676b5fb8d-mlg9k   1/1     Running   0          4m21s   10.129.0.9    kni1-master-1.example.com   <none>           <none>
hello-node-7676b5fb8d-ngdlr   1/1     Running   0          2m29s   10.131.0.6    kni1-worker-0.example.com   <none>           <none>

$ for pod in $(oc get po -o name); do oc rsh $pod cat /etc/hostname; done
error: unable to upgrade connection: error dialing backend: dial tcp 172.22.0.97:10010: connect: no route to host
hello-node-7676b5fb8d-g76bk
hello-node-7676b5fb8d-mlg9k
hello-node-7676b5fb8d-ngdlr
```

Master:
```
apiVersion: v1
kind: Node
metadata:
  annotations:
    machineconfiguration.openshift.io/currentConfig: rendered-master-4ffe5c56a8efc1106cd575b12e9d2414
    machineconfiguration.openshift.io/desiredConfig: rendered-master-4ffe5c56a8efc1106cd575b12e9d2414
    machineconfiguration.openshift.io/reason: ""
    machineconfiguration.openshift.io/state: Done
    volumes.kubernetes.io/controller-managed-attach-detach: "true"
  creationTimestamp: "2019-11-26T12:49:04Z"
  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/os: linux
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: kni1-master-0.example.com
    kubernetes.io/os: linux
    node-role.kubernetes.io/master: ""
    node.openshift.io/os_id: rhcos
  name: kni1-master-0.example.com
  resourceVersion: "72506"
  selfLink: /api/v1/nodes/kni1-master-0.example.com
  uid: a8adb58f-5349-475c-9567-5b8e061e82e3
spec: {}
status:
  addresses:
  - address: 10.19.138.11
    type: InternalIP
  - address: kni1-master-0.example.com
    type: Hostname
  allocatable:
    cpu: 55500m
    ephemeral-storage: "268925769693"
    hugepages-1Gi: "0"
    hugepages-2Mi: "0"
    memory: 263427296Ki
    pods: "250"
  capacity:
    cpu: "56"
    ephemeral-storage: 284964Mi
    hugepages-1Gi: "0"
    hugepages-2Mi: "0"
    memory: 264041696Ki
    pods: "250"
  conditions:
  - lastHeartbeatTime: "2019-11-26T16:35:19Z"
    lastTransitionTime: "2019-11-26T12:49:02Z"
    message: kubelet has sufficient memory available
    reason: KubeletHasSufficientMemory
    status: "False"
    type: MemoryPressure
  - lastHeartbeatTime: "2019-11-26T16:35:19Z"
    lastTransitionTime: "2019-11-26T12:49:02Z"
    message: kubelet has no disk pressure
    reason: KubeletHasNoDiskPressure
    status: "False"
    type: DiskPressure
  - lastHeartbeatTime: "2019-11-26T16:35:19Z"
    lastTransitionTime: "2019-11-26T12:49:02Z"
    message: kubelet has sufficient PID available
    reason: KubeletHasSufficientPID
    status: "False"
    type: PIDPressure
  - lastHeartbeatTime: "2019-11-26T16:35:19Z"
    lastTransitionTime: "2019-11-26T12:53:33Z"
    message: kubelet is posting ready status
    reason: KubeletReady
    status: "True"
    type: Ready
  daemonEndpoints:
    kubeletEndpoint:
      Port: 10250
  images:
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:3deab78351d386f4dab0f7eda97c4bcd4eadeee7e95cd49de6c5a1a7e43c11c6
    - <none>:<none>
    sizeBytes: 829293174
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ac936c0b0bb85e32c4473ebd98151cc1f82b070dd2078cd773349ef7cd490e4b
    - <none>:<none>
    sizeBytes: 822744600
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e9b215a75d0bfddc9e92bd6e20edc6bee372e373eeb13aad7ba3994ae2789319
    - <none>:<none>
    sizeBytes: 804962263
  - names:
    - gcr.io/hello-minikube-zero-install/hello-node@sha256:9cf82733f7278ae7ae899d432f8d3b3bb0fcb54e673c67496a9f76bb58f30a1c
    - gcr.io/hello-minikube-zero-install/hello-node:latest
    sizeBytes: 676724675
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:8f5fed3267b8547cebdccbf954d71fb599a7842e2b02842e397f0652144692ea
    - <none>:<none>
    sizeBytes: 474546779
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:272de9d1dfb5e665ac02305f629c3ef22b92c2db0b33d19d14d2ee7b5c2b5fb5
    - <none>:<none>
    sizeBytes: 467733144
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9fd355d207a75d6944223c315e3dee803fbacf071912359909f0d443a5e6efec
    - <none>:<none>
    sizeBytes: 459530452
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:081d9cee23ea6840fb3e2cc9a3406eddb6f4d20583c5fd2e4915dd534635d435
    - <none>:<none>
    sizeBytes: 447876184
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:4a439c4a128260accac47c791bed2a318f95bdd17d93b5903ab7f8780ef99baf
    - <none>:<none>
    sizeBytes: 441937100
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7882fa3059abf330a1eaa41cebc2751e4a0e18c573907a1cb69ea55c3a9840dc
    - <none>:<none>
    sizeBytes: 435367910
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:46fe9d950df6362b45fd3d42e53a6c3ccc903b641641267b87a1683d9ec427dc
    - <none>:<none>
    sizeBytes: 420481319
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b3551af984766677f22b69ec8375ab08b8f2683d362a6a909f3356f4d88216fd
    - <none>:<none>
    sizeBytes: 409864838
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:11b4030320f7198101f598d235ba74d5cdb425b5178e2c2b39cf2e642039caa0
    - <none>:<none>
    sizeBytes: 384683636
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fbffb2af5f428b842d95d7453c17fc38e72ee3e3e7bb9450ffb3fd31b6ad642c
    - <none>:<none>
    sizeBytes: 372617232
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:41713c22f13c668237557ef9825aa04441fc89fb6dcc1898545b44d003f08039
    - <none>:<none>
    sizeBytes: 362093770
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:3effc05bdafc82c61f5d71cd67b4132f6b7896a3a839698e8b4db334ca8d4635
    - <none>:<none>
    sizeBytes: 341180110
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0a64632759b1c3ea210a8c7737199a5d878c4670f81e783d6e3960d203e634fc
    - <none>:<none>
    sizeBytes: 337893506
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:690ac13b6d32bcba32d2ba54388f7ca5f4c5ec1ae435ec379e365ecff788bc00
    - <none>:<none>
    sizeBytes: 335971029
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2266f58c42e0591b9862bcb76bd61705a07387cde792b6c04dbb462691ebe1e1
    - <none>:<none>
    sizeBytes: 332917424
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:51ee967e0dd8500d9245f8e99d6d01ac13bc8ffcfb89a6c5385ff9d2edd72bc6
    - <none>:<none>
    sizeBytes: 332808216
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dd6b7b640576ba10f01b67c96a3473cdb17304907e7d5ab6291f4b8bf06051cc
    - <none>:<none>
    sizeBytes: 332253427
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cf01bfa6e486cba9230f747cfc50a70a620dbdf4a1600eb8a4708f8fca5f67be
    - <none>:<none>
    sizeBytes: 332008144
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:1320666f8ce8aac6a4381c09a7bff52788dc8ea246188f7704259503f65e2797
    - <none>:<none>
    sizeBytes: 331255485
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ee3388d1c6c34da6ab5e622173f3a84f8e0961b422aaf4667b220f24007180b0
    - <none>:<none>
    sizeBytes: 329672324
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d374cec344d7e4731e01b18254d310de2bafd650fb6db31f126a1408988fc72b
    - <none>:<none>
    sizeBytes: 329407174
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ca9db897672c793cb8254a646eabefed5aceddac7bcce03dffb33a30cadbf6e2
    - <none>:<none>
    sizeBytes: 317559589
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7d6e299c03ab3d9c1b4749ad8943b1264194d1896477736a97c8f42c6f49dc64
    - <none>:<none>
    sizeBytes: 315244108
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:459419e2b363c1ae7692178eba24d0270bb769bc8c323538658879351bc5360e
    - <none>:<none>
    sizeBytes: 315054604
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e800d4990adacb5ba90e7c6a9029bf1064a2475aec741e2c8f76eb9f6f5c6687
    - <none>:<none>
    sizeBytes: 313697398
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e8d7a27c283c0b494df00435077ad5b9e4959a6bd39846e60022b56bfc602c24
    - <none>:<none>
    sizeBytes: 311855135
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a1ed7ff34655c46942c1add61399123ee76899dc9f3058cae834a0660be4a8a3
    - <none>:<none>
    sizeBytes: 311542455
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0a08589814dae86452cffb4a6200354cae817489a6aa61665140b65bf538ed29
    - <none>:<none>
    sizeBytes: 310852279
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:035068065e2ea84733a7f79c868072f35e01e06034004dd84f4affaffd5af756
    - <none>:<none>
    sizeBytes: 310722203
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:346b7b3edf5d849cafedccc69c53ed436aecb4acea02a63729323c0161197e8a
    - <none>:<none>
    sizeBytes: 310440173
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a12024194b9b1210263b792d9edf0fa00cc2c067175682c381764b57839e4096
    - <none>:<none>
    sizeBytes: 308410123
  - names:
    - quay.io/openshift-release-dev/ocp-release-nightly@sha256:eda64817904da6dfae9bf008408c45e1cb9ccf79801c441f3608c1b969894843
    - <none>:<none>
    sizeBytes: 308268103
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:92f5a496a90c0327b0da57ce7c78c73d179047b5fb3c17a5e3348361240b43ba
    - <none>:<none>
    sizeBytes: 308267074
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0427781b4ca4e95552345375805e2f952de3f24d57844e85d8b1283d60b952e3
    - <none>:<none>
    sizeBytes: 307982519
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:234cb579f88f53781e547f202152eb273cf569073818fdeb4ba7cd7086071065
    - <none>:<none>
    sizeBytes: 306447854
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:406a01ca466294b55b2c5709ed40d3968223cd312b5d7197fa29e2ee9cc683ed
    - <none>:<none>
    sizeBytes: 305212973
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7e071a72ea7bcd10016fd5c3f0a5e15ebfcbbaab0c968fb46f454841e28dbb73
    - <none>:<none>
    sizeBytes: 305138402
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:153d3d5b6d453ce819d7589fa26f0c1a1a1494dbac27b2f69c0ba4ddea9fdde9
    - <none>:<none>
    sizeBytes: 304800963
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e09cf3dfab0431cf2630fe207567b1e7e98b42091ac8d76d90120402211938eb
    - <none>:<none>
    sizeBytes: 301791346
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f7fda3ad53ad3f5ca04121a033c88f0cc963ac2c136ce0d85828a2dfa9ed1186
    - <none>:<none>
    sizeBytes: 300114146
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:442bf9eacee2040d1a2a9928d738b7dd93f77c660c4ffc3e35739c05d940f188
    - <none>:<none>
    sizeBytes: 299408022
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:89806e98d4595f3694da0ea7b506eed1ca868a59f05994e8e42904936441e547
    - <none>:<none>
    sizeBytes: 299046121
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b453abf7560ef2a719375904c6a791fceb2734d4ef144a63cd41c79a15464c8b
    - <none>:<none>
    sizeBytes: 297967111
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0de037e5c55faae3f62389803337328a1982c668d6b4b31e39771e6a6f68f125
    - <none>:<none>
    sizeBytes: 297052180
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f60e08105a10ed79678f1b1d801f10254047a0a8874ff11d75a91ecb2bf15a6e
    - <none>:<none>
    sizeBytes: 295891529
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:284379a1750c7738bdcf814e2f7c926eee092db41d78e0b342def3ae86f7399b
    - <none>:<none>
    sizeBytes: 295108757
  nodeInfo:
    architecture: amd64
    bootID: abb406eb-4d10-472c-802c-c5fb7222e3a9
    containerRuntimeVersion: cri-o://1.16.0-2.rhaos4.3.git8b6bfb5.el8
    kernelVersion: 4.18.0-147.0.3.el8_1.x86_64
    kubeProxyVersion: v1.16.2
    kubeletVersion: v1.16.2
    machineID: e2f59da2fe2d412dbe6432e297391795
    operatingSystem: linux
    osImage: Red Hat Enterprise Linux CoreOS 43.81.201911200632.0 (Ootpa)
    systemUUID: 4c4c4544-0050-3510-8031-b7c04f323832
```

Worker:

```
apiVersion: v1
kind: Node
metadata:
  annotations:
    machine.openshift.io/machine: openshift-machine-api/kni1-worker-0-djhf2
    machineconfiguration.openshift.io/currentConfig: rendered-worker-74581ab8ae83d329501782ea1590d16c
    machineconfiguration.openshift.io/desiredConfig: rendered-worker-74581ab8ae83d329501782ea1590d16c
    machineconfiguration.openshift.io/reason: ""
    machineconfiguration.openshift.io/state: Done
    volumes.kubernetes.io/controller-managed-attach-detach: "true"
  creationTimestamp: "2019-11-26T13:34:40Z"
  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/os: linux
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: kni1-worker-0.example.com
    kubernetes.io/os: linux
    node-role.kubernetes.io/worker: ""
    node.openshift.io/os_id: rhcos
  name: kni1-worker-0.example.com
  resourceVersion: "72746"
  selfLink: /api/v1/nodes/kni1-worker-0.example.com
  uid: ce3922dc-6ee3-4604-a185-aacb7f8c5f47
spec: {}
status:
  addresses:
  - address: 10.19.138.9
    type: InternalIP
  - address: kni1-worker-0.example.com
    type: Hostname
  allocatable:
    cpu: 55500m
    ephemeral-storage: "358918756168"
    hugepages-1Gi: "0"
    hugepages-2Mi: "0"
    memory: 197367060Ki
    pods: "250"
  capacity:
    cpu: "56"
    ephemeral-storage: 380324Mi
    hugepages-1Gi: "0"
    hugepages-2Mi: "0"
    memory: 197981460Ki
    pods: "250"
  conditions:
  - lastHeartbeatTime: "2019-11-26T16:36:10Z"
    lastTransitionTime: "2019-11-26T13:34:39Z"
    message: kubelet has sufficient memory available
    reason: KubeletHasSufficientMemory
    status: "False"
    type: MemoryPressure
  - lastHeartbeatTime: "2019-11-26T16:36:10Z"
    lastTransitionTime: "2019-11-26T13:34:39Z"
    message: kubelet has no disk pressure
    reason: KubeletHasNoDiskPressure
    status: "False"
    type: DiskPressure
  - lastHeartbeatTime: "2019-11-26T16:36:10Z"
    lastTransitionTime: "2019-11-26T13:34:39Z"
    message: kubelet has sufficient PID available
    reason: KubeletHasSufficientPID
    status: "False"
    type: PIDPressure
  - lastHeartbeatTime: "2019-11-26T16:36:10Z"
    lastTransitionTime: "2019-11-26T13:35:19Z"
    message: kubelet is posting ready status
    reason: KubeletReady
    status: "True"
    type: Ready
  daemonEndpoints:
    kubeletEndpoint:
      Port: 10250
  images:
  - names:
    - gcr.io/hello-minikube-zero-install/hello-node@sha256:9cf82733f7278ae7ae899d432f8d3b3bb0fcb54e673c67496a9f76bb58f30a1c
    - gcr.io/hello-minikube-zero-install/hello-node:latest
    sizeBytes: 676724675
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:272de9d1dfb5e665ac02305f629c3ef22b92c2db0b33d19d14d2ee7b5c2b5fb5
    - <none>:<none>
    sizeBytes: 467733144
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:4a439c4a128260accac47c791bed2a318f95bdd17d93b5903ab7f8780ef99baf
    - <none>:<none>
    sizeBytes: 441937100
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7882fa3059abf330a1eaa41cebc2751e4a0e18c573907a1cb69ea55c3a9840dc
    - <none>:<none>
    sizeBytes: 435367910
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fbffb2af5f428b842d95d7453c17fc38e72ee3e3e7bb9450ffb3fd31b6ad642c
    - <none>:<none>
    sizeBytes: 372617232
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:3effc05bdafc82c61f5d71cd67b4132f6b7896a3a839698e8b4db334ca8d4635
    - <none>:<none>
    sizeBytes: 341180110
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:51ee967e0dd8500d9245f8e99d6d01ac13bc8ffcfb89a6c5385ff9d2edd72bc6
    - <none>:<none>
    sizeBytes: 332808216
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ca9db897672c793cb8254a646eabefed5aceddac7bcce03dffb33a30cadbf6e2
    - <none>:<none>
    sizeBytes: 317559589
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7d6e299c03ab3d9c1b4749ad8943b1264194d1896477736a97c8f42c6f49dc64
    - <none>:<none>
    sizeBytes: 315244108
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:459419e2b363c1ae7692178eba24d0270bb769bc8c323538658879351bc5360e
    - <none>:<none>
    sizeBytes: 315054604
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e8d7a27c283c0b494df00435077ad5b9e4959a6bd39846e60022b56bfc602c24
    - <none>:<none>
    sizeBytes: 311855135
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:234cb579f88f53781e547f202152eb273cf569073818fdeb4ba7cd7086071065
    - <none>:<none>
    sizeBytes: 306447854
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:28eea6df6af9a6c613feeb4bfa20a60f631049cd13b85a3fe7eb55b087b5d333
    - <none>:<none>
    sizeBytes: 285868112
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:96f5cf1a7738f82a106cf49ee78b32a7dd16f119807db08905e362160efc3679
    - <none>:<none>
    sizeBytes: 277788312
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2592294b8965d8a767f3a52dc3a1406e8e814e8fb762df0bef941470d39403cc
    - <none>:<none>
    sizeBytes: 271210379
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f512b198e0624420b7b7287c0d98dcda2c59d7523a1611b63d2d640f035e12b9
    - <none>:<none>
    sizeBytes: 264163814
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bda52b0ce2df0c48775c1716b689dfc47c6cd633a0d8bdc82b85d113e3714d6b
    - <none>:<none>
    sizeBytes: 257960046
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c9b4dfd8ed97552e9b372247b6ef8a9e5b729bec4ff1a65059a5a139be9b16fd
    - <none>:<none>
    sizeBytes: 256929413
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cf0556fe112d8c6ebde87f3f2d5d3583df9e67c47ff1305356f9077b40495e52
    - <none>:<none>
    sizeBytes: 255884503
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:33cb4b5424a43b257cf9ecd02358b6dd9ea8036347c137a32a175475ed2afdd3
    - <none>:<none>
    sizeBytes: 250744382
  - names:
    - quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7786fed414aeffb21d16c1d440ef5b8893b46f6238de9d617375642f33997370
    - <none>:<none>
    sizeBytes: 238782787
  nodeInfo:
    architecture: amd64
    bootID: f55603bb-41b3-4324-b7f9-d9dc43186bb1
    containerRuntimeVersion: cri-o://1.16.0-2.rhaos4.3.git8b6bfb5.el8
    kernelVersion: 4.18.0-147.0.3.el8_1.x86_64
    kubeProxyVersion: v1.16.2
    kubeletVersion: v1.16.2
    machineID: b6964f189b87449c82959f36ca3a2b60
    operatingSystem: linux
    osImage: Red Hat Enterprise Linux CoreOS 43.81.201911200632.0 (Ootpa)
    systemUUID: 4c4c4544-0056-5210-805a-c7c04f433432
```

```
$ oc version
Client Version: 4.3.0-0.nightly-2019-11-19-053808
Server Version: 4.3.0-0.nightly-2019-11-25-153929
Kubernetes Version: v1.16.2
```

I've only used the baremetal installer (no dev-scripts) and the following install-config.yaml

```
apiVersion: v1
baseDomain: example.com
networking:
  machineCIDR: 10.1.138.0/24
metadata:
  name: kni1
compute:
- name: worker
  replicas: 3
  platform:
    baremetal: {}
controlPlane:
  name: master
  replicas: 1
  platform:
    baremetal: {}
platform:
  baremetal:
    dnsVIP: 10.1.138.16
    apiVIP: 10.1.138.14
    ingressVIP: 10.1.138.15
    provisioningBridge: provisioning
    externalBridge: baremetal
    hosts:
      - name: kni1-master-0
        role: master
        bmc:
          address: ipmi://10.1.136.22
          username: root
          password: password
        bootMACAddress: 11:22:33:44:55:66
        hardwareProfile: default
      - name: kni1-master-1
        role: worker
        bmc:
          address: ipmi://10.1.136.23
          username: root
          password: password
        bootMACAddress: 11:22:33:44:55:66
        hardwareProfile: default
      - name: kni1-master-2
        role: worker
        bmc:
          address: ipmi://10.1.136.24
          username: root
          password: password
        bootMACAddress: 11:22:33:44:55:66
        hardwareProfile: default
      - name: kni1-worker-0
        role: worker
        bmc:
          address: ipmi://10.1.143.29
          username: root
          password: password
        bootMACAddress: 11:22:33:44:55:66
        hardwareProfile: default
pullSecret: |
```

**Comment by stbenjam** [comment]
This also affects `exec`, maybe same root cause:

```
[core@master-1 ~]$ sudo crictl exec -it 5bec919d78d44 /bin/bash
FATA[0003] execing command in container failed: error sending request: Post http://172.22.0.52:10010/exec/jCghrXiO: dial tcp 172.22.0.52:10010: connect: no route to host 
```

**Comment by stbenjam** [comment]
Will be fixed by https://github.com/cri-o/cri-o/pull/3165

**Comment by stbenjam** [comment]
This should be fixed now, the latest builds in 4.5 have cri-o 1.17.
